modules/ROOT/pages/getting-started.adoc

```diff
@@ -22,11 +22,37 @@ This guide shows you how to:
 * Configure the {org-name} showcase app for your mobile platform (Android, iOS, Cordova or Xamarin).
 * Run the showcase app and make calls to the {keycloak-service} service.
 
-Make sure you satisfy all the requirements listed in the xref:prerequisites[] and that you have a mobile app development environment configured. Some experience with OpenShift administration would also be helpful.
-
 // start installing-mobile-services.adoc[leveloffset=1]
 include::_partials/installing-mobile-services.adoc[leveloffset=1]
 // end installing-mobile-services.adoc[leveloffset=1]
+
+[[provision-mdc]]
+== Provisioning Mobile Developer Console
+
+Mobile Developer Console allows you to:
```

Comment:
This section feels like it should be under a heading `Introduction to Mobile Developer Console`. I landed here from a link in the Mobile Developer Console under Provisioning. Felt a bit jarring to land into text which is describing what it is when I expected instructions to Provision @jhellar @finp